.docker/parameters.yml

```diff
@@ -31,7 +31,13 @@ parameters:
     subject_rewrites: []
     example_rds_articles:
         '16846': https://repro.elifesciences.org/example.html
+        '30274': https://repro.elifesciences.org/example.html
```

Comment:
`example_rds_articles` doesn't need to be expanded; it is the existing feature pointing at https://repro.elifesciences.org/example.html that is invisible where the new RDS approach is enabled